decorator.py

```diff
@@ -0,0 +1,19 @@
+from datetime import datetime
+
+now = datetime.now()
+current_time = now.strftime("%H:%M:%S")
+
+def my_decorator(func):
+    def deco_func(*args):
+        now = datetime.now()
+        current_time = now.strftime("%H:%M:%S")
+        with open('logs.txt', 'a+') as f:  #'a' mode creates a file if it doesn't exist
+            f.write(f"Function name: {func} Call time: {current_time}\n")
+        func(*args)
```

Comment:
Add here `return` to be able to decorate functions that return some value. And add **kwargs as well here and in `deco_func` to work with functions that have named arguments